content/docs/components/divider/theming.mdx

```diff
@@ -3,4 +3,173 @@ id: divider
+
+## Theming utilities
+
+- `defineStyle`: a function used to create style objects.
+- `defineStyleConfig`: a function used to define the style configuration for a single part component.
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+```
+
+## Customizing the default theme
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+
+const thicc = defineStyle({
+    border: '5px solid', // change the appearance of the border
+    borderRadius: 10, // set border radius to 10
+})
+
+export const dividerTheme = defineStyleConfig({
+    variants: { thicc },
+})
+```
+After customizing the divider theme, we can import it in our theme file and add it in the components property:
+
+```js
+import { extendTheme } from '@chakra-ui/react'
+import { dividerTheme } from './components/divider.ts'
+
+export const theme = extendTheme({
+    components: { Divider: dividerTheme },
+})
+```
+
+> This is a crucial step to make sure that any changes that we make to the divider theme are applied.
+
+## Adding a custom size
+
+Let's assume we want to include an extra large divider size. Here's how we can do that:
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+
+const xl = defineStyle({
+    border: "10px solid",
+    borderRadius: 'lg',
+})
+
+export const dividerTheme = defineStyleConfig({
+    sizes: { xl },̥
+})
+
+// Now we can use the new `xl` size
+<Divider size="xl">...</Divider>
+```
+
+Every time you're adding anything new to the theme, you'd need to run the cli command to get proper autocomplete in your IDE. Here's how you can do that:
+
+```bash
+# run it once:
+npx chakra-cli tokens path/to/theme.ts
+
+# or run it in watch mode:
+npx chakra-cli tokens path/to/theme.ts --watch
+```
+
+## Adding a custom variant
+
+Let's assume we want to include a custom branded variant. Here's how we can do that:
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+
+const brandPrimary = defineStyle({
+    border: '3px dashed',
+    borderColor: 'orange.500',
+
+    // let's also provide dark mode alternatives
+    _dark: {
+        border: '3px dashed',
+        borderColor: 'orange.300',
+    }
+})
+
+export const dividerTheme = defineStyleConfig({
+    variants: { brandPrimary },
+})
+
+// Now we can use the new `brandPrimary` variant
+<Divider variant="brandPrimary">...</Divider>
+```
+
+## Using a custom color scheme
+
+Let's assume we want to use our own custom color scale based on our brand. We'd need to define the color scale first in the main theme file:
+
+```js
+import { extendTheme } from '@chakra-ui/react'
+
+export const theme = extendTheme({
+    colors: {
+        brand: {
+            50: '#f7fafc',
+            ...
+            500: '#718096',
+            ...
+            900: '#171923',
+        }
+    }
+})
+```
+
+Then, we can use the custom color scale as the color scheme for the divider:
+
+```jsx
+<Divider colorScheme='brand'>...</Divider>
+```
```

Comment:
This is not going to work. The default variants don't support a `colorScheme`, so adding a color scheme without using a custom variant will not produce different results. You can either remove this block, or specify that if the users would like to have a `colorMode` for their Dividers, they'd need to create a custom variant. Also add examples on how they can achieve that.